control-plane/pkg/reconciler/consumergroup/controller.go

```diff
@@ -378,6 +379,7 @@ func createStatefulSetScheduler(ctx context.Context, c SchedulerConfig, lister s
 		Evictor:              newEvictor(ctx, zap.String("kafka.eventing.knative.dev/component", "evictor")).evict,
 		VPodLister:           lister,
 		NodeLister:           nodeinformer.Get(ctx).Lister(),
+		PodLister:            podinformer.Get(ctx).Lister().Pods(system.Namespace()),
```

Comment:
here we need to pass the `dispatcherPodInformer`

Reply:
:+1: